doc/source/whatsnew/v1.1.3.rst

```diff
@@ -18,6 +18,7 @@ Fixed regressions
 - Fixed regression in :class:`IntegerArray` unary plus and minus operations raising a ``TypeError`` (:issue:`36063`)
 - Fixed regression in :meth:`Series.__getitem__` incorrectly raising when the input was a tuple (:issue:`35534`)
 - Fixed regression in :meth:`Series.__getitem__` incorrectly raising when the input was a frozenset (:issue:`35747`)
+- Fixed regression in :meth:`read_excel` with `engine="odf"` caused ``UnboundLocalError`` in some cases where cells had nested child nodes (:issue:`36122`, and :issue:`35802`)
```

Comment:
double-ticks around engine=odf?

`(:issue:`36122`, and :issue:`35802`)` -> (:issue:`36122`,:issue:`35802`)

Reply:
Thanks @jbrockmendel for the comments. This PR is to sync the whatsnew on master and 1.1.x if I make changes to 1.1.3 (requested above) this will require a backport and with changes to 1.2 whatsnew in this PR an auto backport will fail. will split into 2 PRs.